src/restruct_poc/types.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+namespace oturb {
+
+static constexpr std::size_t LieAlgebraComponents = 6;
+static constexpr std::size_t LieGroupComponents = 7;
+static constexpr std::size_t VectorComponents = 3;
```

Comment:
Let's add a prefix `k` in front of these constant per the [style guide recommendation](https://google.github.io/styleguide/cppguide.html#Constant_Names).

```
static constexpr std::size_t kLieAlgebraComponents = 6;
static constexpr std::size_t kLieGroupComponents = 7;
static constexpr std::size_t kVectorComponents = 3;
```